src/lib/chips/chip-list.ts

```diff
@@ -0,0 +1,55 @@
+import {
+  ChangeDetectionStrategy,
+  Component,
+  ContentChildren,
+  ElementRef,
+  ModuleWithProviders,
+  NgModule,
+  QueryList,
+  ViewEncapsulation
+} from '@angular/core';
+
+import {MdChip} from './chip';
+
+@Component(MdChipList.COMPONENT_CONFIG)
+export class MdChipList {
+
+  static COMPONENT_CONFIG: Component = {
+    moduleId: module.id,
+    selector: 'md-chip-list, [md-chip-list]',
+    template: `<ng-content></ng-content>`,
+    host: {
+      // Properties
+      'tabindex': '0',
+      'role': 'listbox',
+    },
+    queries: {
+      items: new ContentChildren(MdChip)
```

Comment:
I will play with this in the AoT and see what happens, but I realized it's actually not used in the skeleton (I built the skeleton by deleting code from what I showed you before) so I will remove it from this PR and probably have it in my next one.